core/src/main/java/org/infinispan/configuration/cache/JMXStatisticsConfigurationBuilder.java

```diff
@@ -9,6 +7,8 @@
 import org.infinispan.util.logging.Log;
 import org.infinispan.util.logging.LogFactory;
 
+import java.lang.invoke.MethodHandles;
```

Comment:
:-/ I have to set the settings in one way before commit, and IntelliJ IDEA automatically formats all modified files. So I would either change the imports order in Parser72/80 or here.